src/packages/smithy/model/credential_management.smithy

```diff
@@ -115,6 +122,28 @@ structure GetInstallationDataOutput {
     installations: InstallationDataList
 }
 
+structure GetInstallationsInput {}
+
+structure InstallationRecord {
+  appId: Integer
+  installationId: Integer
+  nodeId: String
+  targetType: String
+}
+
+list InstallationRecordList {
+   member: InstallationRecord
+}
+
+map InstallationsRecords {
+    key: String
+    value: InstallationRecordList
+}
```

Comment:
Since we include the app ID in the installation record, what is the use-case or nesting installations within (presumably) the app ID vs just having the bare list of all installations?

ie:

```smithy
structure GetInstallationsOutput {
    installations: InstallationRecordList
}
```

If we want to let users filter by specific Apps, we should accept those as inputs (though as a note, I would be inclined to not offer filters for v0).